agent/job_runner.go

```diff
@@ -100,9 +115,6 @@ type JobRunner struct {
 	// The logger to use
 	logger logger.Logger
 
-	// The registered agent API record running this job
-	agent *api.AgentRegisterResponse
-
```

Comment:
I guess this was not needed?

Reply:
we pulled in the entire register response just to get the interval for how often we should check to see that if the job's been cancelled; i figured we can just pull that in directly